friends/src/components/SubmitNewFriend.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+class SubmitNewFriend extends React.Component {
+    constructor(props) {
+        super(props);
+    }
+
+    submitHandler = event => {
+        event.preventDefault();
+        this.props.addNewFriend();
+        this.props.history.push("/");
+    }
+
+    render() {
+        return (
+            <form onSubmit={this.submitHandler}>
+                <input onChange={this.props.handleChange} name="name" placeholder="name" value={this.props.name} type="text" />
+                <input onChange={this.props.handleChange} name="age" placeholder="age" value={this.props.age} type="number" />
+                <input onChange={this.props.handleChange} name="email" placeholder="email" value={this.props.email} type="email" />
```

Comment:
The inputs look very similar. Could you make a function or component to get the parts that are the same taken care of without so much repetition?